client/my-sites/earn/home.tsx

```diff
@@ -198,35 +201,40 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 					},
 			  }
 			: {
-					text: translate( 'Collect payments' ),
+					text: ctaTitle,
 					action: () => {
 						trackCtaButton( 'recurring-payments' );
 						page( `/earn/payments/${ selectedSiteSlug }` );
 					},
 			  };
-		const hasConnectionTitle = translate( 'Manage payments' );
-		const noConnectionTitle = translate( 'Collect payments' );
-		const title = hasConnectedAccount ? hasConnectionTitle : noConnectionTitle;
+		const title = translate( 'Collect payments' );
 
-		const hasConnectionBody = translate(
-			"Manage your customers and subscribers, or your current subscription options and review the total revenue that you've made from payments."
-		);
-		const noConnectionBody = (
+		const body = (
 			<>
-				{ translate(
-					'Accept one-time and recurring credit card payments for physical products, services, memberships, subscriptions, and donations.'
-				) }
-				{ isFreePlan && <em>{ getAnyPlanNames() }</em> }
+				{ hasConnectedAccount
+					? translate(
+							'Let visitors pay for digital goods and services or make quick, pre-set donations by inserting the Payment Button block.'
+					  )
+					: translate(
+							'Let visitors pay for digital goods and services or make quick, pre-set donations by enabling the Payment Button block.'
+					  ) }
+				<>
+					<br />
+					<em>{ getAnyPlanNames() }</em>
+				</>
 			</>
 		);
-		const body = hasConnectedAccount ? hasConnectionBody : noConnectionBody;
 
-		const learnMoreLink = isFreePlan
+		const learnMoreLink = ! hasConnectedAccount
 			? {
-					url: 'https://wordpress.com/support/recurring-payments/',
+					url: 'https://wordpress.com/payments-donations/',
```

Comment:
This URL looks wrong but isn't. The page is equally about payment buttons and donations.